uninstall.php

```diff
@@ -2,7 +2,7 @@
 defined( 'WP_UNINSTALL_PLUGIN' ) || die( 'Cheatin&#8217; uh?' );
 
 if ( ! defined( 'WP_ROCKET_CACHE_ROOT_PATH' ) ) {
-	define( 'WP_ROCKET_CACHE_ROOT_PATH', WP_CONTENT_DIR . '/cache/' );
+	define( 'WP_ROCKET_CACHE_ROOT_PATH', WP_ROCKET_CACHE_ROOT_PATH );
 }
```

Comment:
that doesn't really make sense: if the constant is not defined, you can't assign a value that doesn't exist

Reply:
Fixed constant code issue.